WordPress/gutenberg
# Remove the list view button from the navigation block

## What?


Closes https://github.com/WordPress/gutenberg/issues/47007, https://github.com/WordPress/gutenberg/issues/47008, https://github.com/WordPress/gutenberg/issues/47009 

The "Test a new "off canvas" editor for navigation block using the block inspector and a tree view of the current menu" experiment introduces a button in the navigation block's toolbar to open the settings sidebar.

The goal of the button is to serve as a way for users to discover there is a tree view of their menu in the settings sidebar.

This PR removes it.

## Why?


[Previously](https://github.com/WordPress/gutenberg/issues/42257#issuecomment-1316833855,) the button had another incarnation which was transformed in #46335 to the current state.

The current state has some issues:

- The button is unique to the navigation block and the action opens the settings sidebar. There is a button to open settings.
- It does not function as a sort of shortcut since it should be a toggle.
- Do we want to couple the block to the interface package?
- Do we need to toggle it on and off based on display state of the settings sidebar?
- Do we need to toggle it off if the new tabs of the inspector are set to something other than list?
- We can't even [open the list view directly](https://github.com/WordPress/gutenberg/pull/46907) yet.

## How?


It undoes the elements introduced in  #46335

## Testing Instructions





0. Have the  "Test a new "off canvas" editor for navigation block using the block inspector and a tree view of the current menu" experiment enabled
1. Insert a navigation block
2. Create a menu or select one
3. Notice the block toolbar does not have [this](https://user-images.githubusercontent.com/846565/211553432-ef13192f-0f48-41b9-a525-80fdd9517a0e.png) button

### Testing Instructions for Keyboard


N/A

## Screenshots or screencast 

N/A
